adapter/src/main/java/org/mifos/chatbot/adapter/handler/loan/NextDueDateHandler.java

```diff
@@ -33,7 +34,7 @@
 @Slf4j
 @Component
 public class NextDueDateHandler extends BaseLoanIntentHandler {
-    private static final String[] INTENT_KEYWORDS = {"Due", "Date"};
+    private static final String[] INTENT_KEYWORDS = {"due_date"};
```

Comment:
potentially add stemming/lemmatization based matching for these intent keywords?

Reply:
same as above.